### PR TITLE
[PATCH v6] changelog: updates for odp v1.19.0.2

### PR DESCRIPTION
**Pull request by Bill-Fischofer-Linaro**
Add updates for ODP v1.19.0.2 (Tiger Moth Service Update 2)

Signed-off-by: Bill Fischofer <bill.fischofer@linaro.org>

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,121 @@
+== OpenDataPlane (1.19.0.2)
+=== Summary of Changes
+ODP v1.19.0.2 is the second service update for the Tiger Moth release. It
+incorporates a number of corrctions and enhancements that further improve the
```
Comment (bogdanPricope) [inline CHANGELOG]: corrections

**Comment by bogdanPricope** [comment]
Reviewed-by: Bogdan Pricope <bogdan.pricope@linaro.org>

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,121 @@
...
+==== https://bugs.linaro.org/show_bug.cgi?id=3879[Bug 3879]
+example/l2fwd_simple fails on some systems when using 1GB huge pages
+
+=== Unnumbered Bug Fixes
+* Corrected the handling of timeout events in the scalable scheduler.
+* Fixed IPsec link order to streamline builds.
+
+=== Known Issues
+==== https://bugs.linaro.org/show_bug.cgi?id=3774[Bug 3774]
+Shmem validation test runs indefinitely with 1GB huge pages
```
Comment (joseppc) [inline CHANGELOG]: Only "with large amounts of 1GB huge pages", I'd say.
When we configured the system with 28 huge pages of 1 GB in size, the tests ran fine, and we were unable to reproduce this issue.

Comment (Bill-Fischofer-Linaro) [reply CHANGELOG]: The changelog simply gives the Bug number and their titles. Those details are (or should be) in bugzilla itself.

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v4
* Spelling corrections as reported by @bogdanPricope and spellcheck.

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v5:
* Updated to include anticipated merge of PR #627 and PR #633.

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v5:
* Remaining M4 macros are not GPL, so remove sentence that misidentified them as such.

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**